### PR TITLE
Color Replaced

### PR DESCRIPTION
**Pull request by Aakash2408**
## Fixes

Fixes #[#62 ] by @[@panchovm]

## Description

Color change from  DarkTurquoise with ForestGreen across all site.

## Technical details

    rgb(34,139,34):This is the value of Forest Green

## Screenshots

Screenshot from 2020-08-18 08-21-10

## Checklist
 

- [x] My pull request targets the *develop* branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

### CODE REVIEW

File: legal_db/static/styles.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $color-tomato: rgb(237, 89, 47);
 $color-dark-turquoise: rgb(5, 181, 218);
```
Comment (kgodey) [inline legal_db/static/styles.scss:2]: please remove this since it's no longer used

**Comment by kgodey** [comment]
@Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.

Please fix the merge conflicts by merging the latest `develop` into your branch.

File: legal_db/static/styles.scss (hunk 1 above)
Comment (Aakash2408) [reply legal_db/static/styles.scss:2]: ok i wil remove it

**Comment by Aakash2408** [comment]
> @Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.
> 
> Please fix the merge conflicts by merging the latest `develop` into your branch.
ok I will do it

**Comment by Aakash2408** [comment]
> @Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.
> 
> Please fix the merge conflicts by merging the latest `develop` into your branch.

At that time I just switched  to develop branch, created my new branch and made changes into that and made a pull request to merge that into the main branch

**Comment by Aakash2408** [comment]
@kgodey  sorry there were major issues on my locals for rebasing and merge. So I opened another pull request kindly check #72 .

**Pull request closed without merging**